esp-wifi/src/ble/controller/mod.rs

```diff
@@ -83,7 +83,7 @@ impl Write for BleConnector<'_> {
 }
 
 /// Async Interface
-#[cfg(feature = "async")]
+
```

Comment:
Blank lines after doc comments trigger clippy errors on nightly

Reply:
Thanks! I figured rustfmt would pick these up, but it must have missed a few some how.